artifacts/scripts/publish_api.sh

```diff
@@ -35,9 +35,9 @@ set -o nounset
 set -o pipefail
 
 if [ ! $# -eq 4 ]; then
-    echo "usage: $0 src_branch dst_branch dependent_k8s_repos kubernetes_remote"
+    echo "usage: $0 src_branch dst_branch dependent_k8s_repos kubernetes_remote subdirectory source_repo_org source_repo_name" 
     exit 1
 fi
 
 SCRIPT_DIR=$(dirname "${BASH_SOURCE}")
-"${SCRIPT_DIR}"/publish_template.sh "api" "${1}" "${2}" "${3}" "${4}" "true"
+"${SCRIPT_DIR}"/publish_template.sh "api" "${1}" "${2}" "${3}" "${4}" "${5}" "${6}" "${7}" "true"
```

Comment:
Note to everybody else: we plan to remove the publish scripts in a follow-up and only move the smoke tests into the rules struct. This ugly argument list will go away.

Reply:
i should actually start an issue to track follow ups

Reply:
:+1: